howso/direct/client.py

```diff
@@ -2643,23 +2643,33 @@ def react(  # noqa: C901
                 defined in feature attributes.
 
             - robust_computation: bool, optional
-                Default is False, uses leave-one-out for features (or cases,
-                as needed) for all relevant computations. If True, uses
-                uniform sampling from the power set of all combinations of
+                Deprecated. If specified, will overwrite the value of both
```

Comment:
A user that is still using `robust_computation` has no expectation to get `robust_influences` set too. If this user wants to also get `robust_influences`, they should adjust their code to stop using this one and start using `robust_residuals` and `robust_influences` in accordance with their needs.

Reply:
I understand better that your approach is great.